demo/full/scripts/modules/player/catchUp.js

```diff
@@ -11,7 +11,7 @@ import fromPlayerEvent from "./fromPlayerEvent";
 
 // Distance from live edge we try to reach when the catching up button
 // is enabled.
-const LIVE_GAP_GOAL_WHEN_CATCHING_UP = 3;
+const LIVE_GAP_GOAL_WHEN_CATCHING_UP = 3.5;
```

Comment:
This is the value in the demo we will try to reach when we're behind the live yet "sticking to the live edge".
Adding 500 milliseconds to it greatly improve the performance of our algorithm